File:  PYTHON/logica.py
def eh_par(n):
    return n % 2 == 0

numero = int(input("Digite um número: "))

if eh_par(numero):
    print("O número é par.")
else:
    print("O número é ímpar.")

print(f"Contando até {numero}:")
for i in range(1, numero + 1):
    print(i)

File:  JavaScript/logica.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

function ehPar(n) {
    return n % 2 === 0;
}

readline.question('Digite um número: ', input => {
    const numero = parseInt(input);

    if (ehPar(numero)) {
        console.log("O número é par.");
    } else {
        console.log("O número é ímpar.");
    }

    console.log(`Contando até ${numero}:`);
    for (let i = 1; i <= numero; i++) {
        console.log(i);
    }

    readline.close();
});
